client/site-profiler/hooks/use-domain-param.ts

```diff
@@ -7,19 +7,25 @@ import {
 import validateDomain from 'calypso/site-profiler/utils/validate-domain';
 import isSpecialDomain, { prepareSpecialDomain } from '../utils/is-special-domain';
 
-export default function useDomainParam( value: string = '' ) {
+export default function useDomainParam( value = '' ) {
 	const [ domain, setDomain ] = useState( value );
 	const [ isValid, setIsValid ] = useState< undefined | boolean >();
 	const [ isSpecial, setIsSpecial ] = useState( isSpecialDomain( value ) );
 	const [ category, setCategory ] = useState< SPECIAL_DOMAIN_CATEGORY >();
 	const [ readyForDataFetch, setReadyForDataFetch ] = useState( false );
 
 	useEffect( () => {
-		const preparedDomain = prepareDomain( value, isSpecial );
-
+		const isSpecialDomainValue = isSpecialDomain( value );
+		const validateDomainValue = validateDomain( value );
```

Comment:
Can we follow the same naming convention here?
If there is the `isSpecialDomainValue`, validateDomainValue could be `isValidDomainValue`.

Reply:
Nice suggestions, changed!